src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -12,8 +12,9 @@
     "name" : "Benson Meier",
     "phone" : "98765432",
     "lessonPlan" : "Trigonometry",
-    "homework": [ "Science worksheet", "math asst" ],
+    "homework": [ "Science worksheet", "Math asst" ],
```

Comment:
You don't have to change this, but if I'm not wrong Christel already made it such that the fields are no longer case sensitive.

Reply:
yeap! changed it i the midst of debugging some test cases, fyi the issue was not with this